lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -184,31 +182,36 @@ sub call_rpc {
 
     foreach my $hook ($before_call_hooks->@*) { $hook->($c, $req_storage) }
 
+    my ($msg_type, $request_data) = $self->_prepare_request_data($c, $req_storage);
```

Comment:
Why is this moved into the foreach?

Reply:
it's not moved into foreach , but moved after the loop.. 
Based on JSONRPC implementation it's not quite correct  we create jsonrpc request before calling `before_call_hooks`, which could affect request parametrs.
probably better to be consistent in this ordering with current implementation.

https://github.com/binary-com/perl-Mojo-WebSocketProxy/blob/master/lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm#L100-L107

Reply:
Oops, sorry, didn't expect perltidy to throw everything into one line there. :)